### ISSUE TITLE
Webots Help menu refactor

### ISSUE THREAD

**Issue by fabienrohrer** [cyberbotics/webots#873]
The Webots help menu does not reflect anymore our public services state.

<img width="417" alt="Capture d’écran 2019-09-10 à 09 16 58" src="https://user-images.githubusercontent.com/866788/64592647-b968ce80-d3ac-11e9-933f-bcb817d445f3.png">

Proposal to refactor the 3 last items:

```
GitHub repository...  # => https://github.com/cyberbotics/webots
Cyberbotics website...  # => https://www.cyberbotics.com
Report an issue...   # => https://github.com/cyberbotics/webots/issues/new/choose
Follow us >
- Subscribe to the Newsletter... # => https://www.cyberbotics.com/news/subscribe.php
- Join the Discord chanel... # => https://discordapp.com/invite/nTWbN9m
- Twitter... # => https://twitter.com/webots
- youTube... # => http://www.youtube.com/user/cyberboticswebots 
```

**Comment by DavidMansolino** [comment]
We should indeed have done this a while ago !

**Comment by fabienrohrer** [comment]
And the `Linked In` link..

**Comment by omichel** [comment]
I would keep the link to the support ticket as it is nice to the users who paid for a Premier service and it may encourage more people to purchase a Premier service.

**Comment by fabienrohrer** [comment]
Yes, it seems better. Maybe I could add a `(Premier Service)` suffix in order to warn the regular users they have to login.

**Comment by fabienrohrer** [comment]
Fixed in #874
